deployment/mcad-controller/crds/mcad.ibm.com_appwrappers.yaml

```diff
@@ -889,6 +889,38 @@ spec:
                   (is this different from the MinAvailable from JobStatus)
                 format: int32
                 type: integer
+              itemCompletionStatus:
+                properties:
+                  genericItems:
+                    items:
+                      properties:
+                        condition:
+                          type: string
+                        gvk:
+                          description: Group Version Kind 
+                          properties:
+                            Group:
+                              type: string
+                            Kind:
+                              type: string
+                            Version:
+                              type: string
+                          required:
+                          - Group
+                          - Kind
+                          - Version
+                          type: object  
+                        name:
+                          type: string
+                        namespace:
+                          type: string
+                      required:
+                      - condition
+                      - name
+                      - namespace
```

Comment:
Some objects may not be namespaced so this field may not exist.  An example is a `namespace` and `clusterrolebinding`.  While those objects are not specific to a completion status there are objects that a may fit the bill and we would want to support that model.